evm/src/fuzz/invariant/mod.rs

```diff
@@ -56,7 +56,7 @@ pub fn assert_invariants(
     for func in &invariant_contract.invariant_functions {
         let mut call_result = executor
             .call_raw(
-                CALLER,
+                default_caller(&Network::Mainnet),
```

Comment:
In all cases where we call `default_caller(network)` there is hardocded `Network::Mainnet` so in the end it is still the same if we hardcoded the mainnet address, we need to get the Network we are on from the context of the function call.

Reply:
These parts are done